### ISSUE TITLE
Library Vulnerabilities

### ISSUE THREAD

**Issue by alonrks** [brendanashworth/generate-password#69]
According to White Source, **underscore** library version 1.7.0 has CVE-2021-23358 security vulnerability.
Is it possible to upgrade to version **underscore** - 1.12.1,1.13.0-2 and release new generate-password version?

**Comment by brendanashworth** [comment]
@alonrks I don't think this is an actual attack vector because underscore is only used in the testing suite. Nonetheless, would you like to submit a PR upgrading it, and I'll release a new version?

**Comment by alonrks** [comment]
Thanks Brendan, could you grant me access please?

**Comment by alonrks** [comment]
@brendanashworth can you please review?
